app/models/login.rb

```diff
@@ -0,0 +1,15 @@
+class Login < ApplicationRecord
+    validates :email, presence: true, uniqueness: true
+    validates :password, presence: true, length: { minimum: 10 }
+    validates :name, presence: true, length: { minimum: 10 }
```

Comment:
https://medium.com/nerd-for-tech/introduction-to-ruby-on-rails-and-how-mvc-works-c56dff61dce5